### ISSUE TITLE
OCR boxes overlapping each other

### ISSUE THREAD

**Issue by LauraErhard** [locdb/locdb-frend#340]
If there is a huge box over other boxes, I can't click on the boxes below.
![huge box](https://user-images.githubusercontent.com/32642653/43761168-9a88ee2c-9a24-11e8-9597-9bec0503582b.png)
I tried clicking through the different boxes with the "j" and "k" shortcuts and this seems to work. However I can't click through them neatly from top to bottom, but the active box switches rather randomly. While drawing the boxes for @rtahseen I noticed, that the boxes are not in order (top to bottom) even then, is this a result? Would it be possible to arrange them in the right order?

**Comment by rtahseen** [comment]
This issue was reported last week during the web meeting. I am working on it right now to avoid those big boxes and also to send back results arranged in right order (top to bottom).

**Comment by lgalke** [comment]
I depicted the problem in #341, @rtahseen maybe this resolves the problem already, so if you struggle to reproduce the problem on your data, it is likely that it is caused by #341. #341 is a front-ent problem but we cannot fix it without adaptions in the back-end and probably also the OCR component.

**Comment by lgalke** [comment]
@rtahseen the inconsistency issues in the front-end are now resolved. The remaining overlaps are to-be-fixed by you. ;)

**Comment by lgalke** [comment]
**Update** we found that the main cause for overlapping boxes are multiple namer and detector strategies. We intend to build specific filtering mechanisms in the front-end, the progress is tracked in PR #369.

**Comment by lgalke** [comment]
We included the filtering mechanism for `Detector` and `Namer` attributes.

**Comment by LauraErhard** [comment]
I assumed the testing label indicates that I should test this but I just looked at the filter and I can see them but I can't choose anything different then `all` for both `Detector` and `Namer`. I looked at #369 and I'm not sure the whole progess is finished?!

**Comment by lgalke** [comment]
Yes it's considered finished. The guess is that most current resources are ingested before the attributes `namer` and `detectors` were provided. That's why we also included the year (primarily for testing purposes). The behaviour should be the same for the other attributes, if there are values in the data, they would be automatically available in the selection. It'd quite interesting to check whether it works for a newly ingested resource.
